### PR TITLE
Remove MVCC import

### PR DESCRIPTION
**Pull request by itaiad200**

### CODE REVIEW

File: onboard/import.go [hunk 1]
```diff
@@ -37,45 +36,37 @@ type Config struct {
...
+	if err := res.CatalogActions.InitBranch(ctx); err != nil {
 		return nil, fmt.Errorf("failed to get previous commit: %w", err)
 	}
-	res.previousCommit = previousCommit
-	shouldSort := res.previousCommit != nil || res.rocks
-	res.inventory, err = config.InventoryGenerator.GenerateInventory(ctx, logger, config.InventoryURL, shouldSort, config.KeyPrefixes)
+
+	res.inventory, err = config.InventoryGenerator.GenerateInventory(ctx, logger, config.InventoryURL, true, config.KeyPrefixes)
 	res.prefixes = config.KeyPrefixes
 	if !res.validatePrefixes() {
```
Comment (johnnyaug) [inline onboard/import.go:71]: Looks like this wouldn't work anymore (not sure if due to this PR or a previous one). Under the Importer struct, we still have a previousCommit member which is used in this method, but it is never assigned.

**Review by johnnyaug — CHANGES_REQUESTED** [review]

File: onboard/import.go (hunk 1 above)
Comment (itaiad200) [reply onboard/import.go:71]: This one. MVCC used that feature.

Comment (johnnyaug) [reply onboard/import.go:71]: But we still want to validate the prefixes, no?

Comment (itaiad200) [reply onboard/import.go]: function name was misleading, it validated that the old and new prefixes match.
That doesn't matter anymore.

**Review by johnnyaug — APPROVED** [review]

**Pull request merged**